services/imagery/src/image-store.js

```diff
@@ -180,10 +189,13 @@ export default class ImageStore extends EventEmitter {
     await this._withDb(async (db) => {
       if (id === undefined) {
         id = (await db.run('INSERT INTO images DEFAULT VALUES')).lastID;
+        // Can we simplify this to just one insert statement?
+        await db.run('UPDATE images SET size = ? WHERE id = ?',
+          [metadata.size, id]);
       } else {
-        await db.run('INSERT INTO images(id) VALUES (?)', id);
+        await db.run('INSERT INTO images(id, size) VALUES (?, ?)',
+          [id, metadata.size]);
```

Comment:
The metadata is currently not set up to pass in the size. You'll need to edit the [base backend](https://github.com/uavaustin/orchestra/blob/master/services/imagery/src/backends/camera-base-backend.js) to calculate the size of each photo taken, and update the [Protobuf message](https://github.com/uavaustin/orchestra/blob/master/services/common/messages/imagery.proto) to include a size field.